cvat-canvas3d/src/typescript/canvas3dView.ts

```diff
@@ -0,0 +1,213 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { Canvas3dController } from './canvas3dController';
+import { Listener, Master } from './master';
+
+import consts from './consts';
+import * as THREE from 'three';
+import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader.js';
+
+import { Canvas3dModel, UpdateReasons, ActiveElement, Mode, Configuration } from './canvas3dModel';
+
+export interface Canvas3dView {
+    html(): HTMLDivElement;
+    render(): any;
+}
+
+export class Canvas3dViewImpl implements Canvas3dView, Listener {
+    private loadingAnimation: SVGSVGElement;
+    private text: SVGSVGElement;
+    private adoptedText: SVG.Container;
```

Comment:
Cannot find namespace 'SVG'.ts(2503)
Because svg.js is not used in canvas3d

Reply:
Resolved